extensions/amp-analytics/0.1/events.js

```diff
@@ -306,4 +302,107 @@ export class IniLoadTracker extends EventTracker {
     });
     return NO_UNLISTEN;
   }
+
+  /**
+   * @return {!Promise}
+   */
+  getRootSignal() {
```

Comment:
No. This is actually root signal, not root element. I.e. "analytics root" to be more precise. For some (most) triggers root element falls back to root. This is similar to `click` event where you can't really click on a document, but you can click on the root element, but you can equate them for most reasons.